vector/src/main/res/layout/activity_tchap_room_creation.xml

```diff
@@ -74,6 +74,32 @@
                 </android.support.design.widget.TextInputLayout>
             </LinearLayout>
 
+            <LinearLayout
```

Comment:
the switch can have a text, so I think this can be replaced by simple Switch

Reply:
I confirm switch have a text, but I failed to reproduce the current layout with this text. So I plan to keep the TextView here ;)